packages/perseus-editor/src/components/graph-settings.tsx

```diff
@@ -317,7 +358,8 @@ const GraphSettings = createReactClass({
         const scale = Util.scaleFromExtent(ranges[i], this.props.box[i]);
         if (this.validRange(ranges[i]) === true) {
             step[i] = Util.tickStepFromExtent(ranges[i], this.props.box[i]);
-            gridStep[i] = Util.gridStepFromTickStep(step[i], scale);
+            // don't like this use of as. :(
```

Comment:
Yeah. I guess it's that `gridStepFromTickStep` could return `null | undefined`. This is the way it is already so I think it's ok as we're kinda acknowledging that the old code didn't take that into account.